### ISSUE TITLE
Dynamic imports break extract

### ISSUE THREAD

**Issue by edwardsCam** [format-message/format-message#156]
format-message-cli is currently incompatible with webpack's codesplitting feature using dynamic imports. We use this feature in my project for lazyloading modules, but right now we can't run `format-message extract` as it fails like so:

```
SyntaxError: ui/packages/career-app/src/routing.js: Unexpected token (37:23)
  35 |     path: '/career/goals/:goalId',
  36 |     behavior: {
> 37 |       component: () => import('./screens/Goal.js'),
     |                        ^

```

This can be fixed by adding the babel plugin to allow that syntax.
Pull request: #155

**Comment by thetalecrafter** [comment]
If you are using a `.babelrc` file, the babel api used by the extract cli should pick that up, and apply any plugins you use. I have a work project that is successfully extracting from code that includes dynamic `import()` calls.

However, given there is already a list of plugins that extract uses by default, I think it's great to add this one.

**Comment by thetalecrafter** [comment]
Fixed in v5.2.4
